internal/engine/k8srollout/podmonitor_test.go

```diff
@@ -113,3 +115,22 @@ func (s *testStore) Dispatch(action store.Action) {
 		_, _ = s.out.Write(logAction.Message())
 	}
 }
+
+func assertSnapshot(t *testing.T, output string) {
```

Comment:
I really like this Golden Master testing method that @jazzdan implemented a while back, because it makes testing log output SO much less painful. Originally it was just pipeline_state_test.go, then I added it to logstore_test.go and now here. So now this ``assertSnapshot` function is duplicated in three places. 

Should / can this be more DRY?